### DISCUSSION TITLE
申请退款，无法使用

### DISCUSSION THREAD

**Discussion by CumanMercenary** [getlantern/lantern-forum-cn#1048] [General]
买了一年的专业版，结果是windows上什么软件进去都太慢了，连进去个google翻译网页都给你卡半天，YouTube看视频用720p看视频都不稳定，twitch甚至直播都不能看。手机上也是，除了浏览器是正常运行的，其它什么软件都进不去。请给我退款
![62B58E1726F099551E3E6FDFE22E9145](https://github.com/user-attachments/assets/675d43ad-a8d7-40a4-8fe0-75b6579d34a1)
![E2$~ B9D_G81F5IKM_J4G 7](https://github.com/user-attachments/assets/f8b92cb6-a1eb-40bb-b7d1-e542784faeaa)

**Comment by MaxMKMax** [comment]
@CumanMercenary 您好. 请您也提供我账号邮箱. 谢谢.

**Reply by CumanMercenary** [reply]
2665242009@qq.com

**Reply by CumanMercenary** [reply]
请给我退款谢谢

**Comment by MaxMKMax** [comment]
@CumanMercenary 
您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

在帮您退款之前可以帮一个忙吗? 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by CumanMercenary** [reply]
code:79678842
